### DISCUSSION TITLE
Disable timeline hover feature

### DISCUSSION THREAD

**Discussion by antonioaltamura** [cypress-io/cypress#9650] [Cypress Cloud]
I see the point of the "timeline" feature, but in my case, it is quite annoying: as soon as you **hover** on an entry in the Cypress log, all the previous actions are repeated.
This means every time I pass the mouse over an entry there (by mistake) I flood the server with dozens of unnecessary calls.
Is there a way to disable the hover action?

**Comment by namuol** [comment]
Try setting `numTestsKeptInMemory: 0` in your cypress configuration. This should prevent "DOM snapshots" from being recorded, thereby preventing the issue from happening.

You can also configure this per test with a configuration object passed as a second argument; it works with `context`, `describe`, `it`, and maybe some other things:

```ts
context('foo bar baz', {numTestsKeptInMemory: 0}, () => {
  ...
});
```
